Analysis/include/AnaTuple.h

```diff
@@ -157,7 +166,9 @@ class AnaTupleWriter {
 
     AnaTupleWriter(const std::string& file_name, Channel channel, bool _runKinFit, bool _runSVfit, bool _allow_calc_svFit);
     ~AnaTupleWriter();
-    void AddEvent(EventInfo& event, const DataIdMap& dataIds, const bool pass_VBF_trigger);
+    void AddEvent(EventInfo& event, const DataIdMap& dataIds, const bool pass_VBF_trigger, const double btag_weight,
```

Comment:
Define `CategoriesFlags` in AnaTuple.h and pass it as an argument to `AddEvent`.